src/Fixer/Phpdoc/PhpdocReadonlyCommentToKeywordFixer.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\Phpdoc;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\DocBlock\DocBlock;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Marcel Behrmann <marcel@behrmann.dev>
+ */
+final class PhpdocReadonlyCommentToKeywordFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     *
+     * Must run before NoExtraBlankLinesFixer, NoTrailingWhitespaceFixer, NoWhitespaceInBlankLineFixer, PhpdocAlignFixer.
+     */
+    public function getPriority(): int
+    {
+        return 4;
+    }
+
+    public function isCandidate(Tokens $tokens): bool
+    {
+        if (!\defined('T_READONLY')) {
+            return false;
+        }
+
+        return $tokens->isTokenKindFound(T_DOC_COMMENT);
+    }
+
+    public function getDefinition(): FixerDefinitionInterface
+    {
+        return new FixerDefinition(
+            'Converts readonly comment to readonly keyword.',
```

Comment:
does it convert to keyword for everything, or only for classes? if only later, it's OK - but we should mention that in description (and in classname)

Reply:
Updated and moved it into the ClassNotation Namespace.